packages/pwa-kit-create-app/assets/bootstrap/js/jest.config.js

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright (c) 2024, Salesforce, Inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+// eslint-disable-next-line @typescript-eslint/no-var-requires
```

Comment:
which eslint config does this conflict with? I wonder if this eslint disable makes sense in a generated project.

If it's only an issue in the `pwa-kit-create-app` project, then we should update the eslint config (?)

Reply:
It shows up as a linting error in both `pwa-kit-create-app` and the generated project `retail-react-app-demo`